src/store/audio.ts

```diff
@@ -525,8 +525,13 @@ export const audioStore = createPartialStore<AudioStoreTypes>({
       { audioKeys }: { audioKeys?: AudioKey[] }
     ) {
       const uniqueAudioKeys = new Set(audioKeys);
-      if (getters.ACTIVE_AUDIO_KEY) {
-        uniqueAudioKeys.add(getters.ACTIVE_AUDIO_KEY);
+      if (
+        getters.ACTIVE_AUDIO_KEY &&
+        !uniqueAudioKeys.has(getters.ACTIVE_AUDIO_KEY)
+      ) {
+        throw new Error(
+          "Assertion failed: audioKeys.includes(getters.ACTIVE_AUDIO_KEY)"
```

Comment:
（ちょっと細かいですが）
エラーメッセージは見た人がどうすればそれのエラーを解消すれば良いのかがわかる形が最高だと思います。今のままだと解決の方針を立てづらいかもしれません。
こんな感じでしょうか。
```suggestion
          "selectedAudioKeys must be include activeAudioKey"
```